src/sentry/monitors/models.py

```diff
@@ -249,6 +249,11 @@ class Monitor(Model):
     Type of monitor. Currently there are only CRON_JOB monitors.
     """
 
+    owner_actor_id = BoundedPositiveIntegerField(null=True, db_index=True)
```

Comment:
We should use `BoundedBigIntegerField` here

Reply:
Ah, got it